### ISSUE TITLE
IR integration: Horn and bell on locomotive

### ISSUE THREAD

**Issue by latibro** [latibro/Automation-Mod#6]
Related to the Immersive Railroading integration

Expanding the locomotive controls with horn and bell

**Comment by latibro** [comment]
Maybe something like

```lua
locomotive.soundHorn()
locomotive.soundBell()
```

**Comment by latibro** [comment]
Would we need more control?

*   Horn
    *   Sound horn for at default set time
    *   Sound horn for a given time (maybe in seconds)
    *   Start horn (sounds the horn until turned off)
    *   Stop horn
*   Bell
    *   Sound bell for at default set time
    *   Sound bell for at given time (maybe in seconds)
    *   Start bell (sounds the bell until turned off)
    *   Stop bell

**Comment by latibro** [comment]
Included in 0.1.0.alpha.4

```lua
locomotive.soundHorn()
locomotive.startBell()
locomotive.stopBell()
```
